packages/cs-parser/src/beacon/parsingOrchestrator.machine.ts

```diff
@@ -141,6 +151,7 @@ export const parsingOrchestratorMachine = createMachine(
 								ip,
 								os: findOsFromMetaLine(metadataLog.blob),
 								host: metadataLog.beacon?.host as Host,
+								shape: Shapes.circle,
```

Comment:
@GoldingAustin I had to add `shape: Shapes.circle` here in `hostMeta` and in `beaconMeta`. Not sure why its marked as required?